Documentation/GettingStarted.md

```diff
@@ -40,45 +40,48 @@ If you press and hold the button near the rear of the iron it will take you into
 
 ## Soldering Mode
 
-When you press the button to enter into the soldering mode, the iron will instantly start to heat up the tip.
+When you press the button to enter the soldering mode, the iron will instantly start to heat up the tip.
 
 The firmware defaults to 320C as the set point for the soldering mode, however on this screen you can enter into the adjustment screen by pressing either button.
 
-Pressing and holding the button near the tip will enter **boost** mode. This allows temporarily overriding the set temperature to a higher (or lower) value. This can be useful as a way to force the tip to a higher temperature to drive more wattage into a large joint when the thermal connection is not ideal.
+Pressing and holding the button near the tip will enter **boost** mode. This allows temporarily override of the set temperature to a higher (or lower) value. This can be useful as a way to force the tip to a higher temperature to drive more wattage into a large joint when the thermal connection is not ideal.
```

Comment:
You have changed the grammar here; you will need to reword the sentence to make more sense since "temporarily override of" doesn't really work

Reply:
fixed